app-admin/pwman3/metadata.xml

```diff
@@ -2,13 +2,11 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 <maintainer type="person">
-	<name>Oz Tiram</name>
-	<email>oz.tiram@gmail.com</email>
+	<email>proxy-maint@gentoo.org</email>
```

Comment:
well, it should also list you first, not only proxy-maint, also please get rid of merge commits and squash existing ones into the single one.

Reply:
@Zlogene should my metadata.xml contain both such as:

```
<maintainer type="person">
	<name>Oz Tiram</name>
	<email>oz.tiram@gmail.com</email>
	<email>proxy-maint@gentoo.org</email>
	<name>Proxy Maintainers</name>
</maintainer>

```

Reply:
@oz123 yes, but one `<maintainer type="...">` tag per "logical maintainer" there are project and person types